docs/metasploit-framework.wiki/dev/Setting-Up-a-Metasploit-Development-Environment.md

```diff
@@ -129,27 +127,103 @@ Congratulations! You have now set up a development environment and the latest ve
 
 ## Optional: Set up the REST API and PostgreSQL database
 
-The following optional section describes how to manually install PostgreSQL and set up the Metasploit database.  Alternatively, use our Omnibus installer which handles this more reliably.
+Installing the REST API and PostgreSQL is optional, and can be done in two ways.
+Recommended is to use the Docker approach, and fairly simple to do once you have docker installed on your
+system, [Docker Desktop][docker-desktop] is recommended, but not mandatory.
+On Linux systems, simply having docker-cli is sufficient.
+
+### Docker Installation
+
+**Make sure, you have docker available on your system: [Docker Installation Guide][docker-installation]**
+
+**Note**: Depending on your environment, these commands might require `sudo`
+
+* Pull the Docker container: `docker pull postgres:14`
+* Start the container:
+
+```bash
+docker run --name "postgres" -p 5432:5432 -e POSTGRESS_PASSWORD="mysecretpassword" -d postgres:14
```

Comment:
Not a blocker; it might be less confusing for newcomers to run this synchronously as well as treating it as a throwaway container

```suggestion
docker run --rm -p 127.0.0.1:5432:5432 -e POSTGRESS_PASSWORD="mysecretpassword" postgres:14
```

Not a blocker for me; but we should probably bind to 127.0.0.1 in the docs either way

Reply:
Hmm, not sure I agree with the approach of setting up the documentation as a throw-away container. 
If you're planning on doing more contributions and development, it's probably easier to retain the container and update with migrations as the work progresses than having to repull every time?

I'd consider this more advanced usage than just setting up a generic postgres container that can be used for multiple projects.